tools/wpt/browser.py

```diff
@@ -17,33 +17,41 @@
 
 uname = platform.uname()
 
-def path(path, exe):
-    path = path.replace("/", os.path.sep)
-    if exe and uname[0] == "Windows":
-        path += ".exe"
-    return path
-
 
 class Browser(object):
     __metaclass__ = ABCMeta
 
     @abstractmethod
     def install(self, dest=None):
+        """Install the browser."""
         return NotImplemented
 
     @abstractmethod
-    def install_webdriver(self):
+    def install_webdriver(self, dest=None):
+        """Install the Webdriver implementation for this browser."""
```

Comment:
WebDriver capitalization here and elsewhere

Reply:
Thanks. I'll grep and fix.